pkg/kube/inject/testdata/inject/proxy-override-runas.yaml.cni.injected

```diff
@@ -138,7 +138,7 @@ spec:
           readOnlyRootFilesystem: true
           runAsGroup: 4321
           runAsNonRoot: true
-          runAsUser: 1234
+          runAsUser: 1337
```

Comment:
Why do we have to change these files?

Reply:
This was included in the cherry-pick from the rebase in 2.5

Reply:
I think this is a new file in 1.20, it was not present on OSSM 2.5.

Reply:
https://github.com/maistra/istio/pull/853/files

Reply:
This file in particular is new in 1.20/2.6, no?

Reply:
Hmm, yes it is new (not in 1.19,2.5)

Reply:
Anyway, it's confusing why we need to change this golden file. The input file explicitly overrides these values and expect them to not be touched: https://github.com/maistra/istio/blob/f1c1c5d98915b343c06f10b189eac6b24e74de23/pkg/kube/inject/testdata/inject/proxy-override-runas.yaml#L19-L22